http://sourceware.org/ml/gdb-cvs/2010-09/msg00077.html

### src/gdb/testsuite/ChangeLog	2010/09/09 20:00:48	1.2443
### src/gdb/testsuite/ChangeLog	2010/09/10 20:29:25	1.2444
## -1,3 +1,8 @@
+2010-09-10  Edjunior Barbosa Machado  <emachado@linux.vnet.ibm.com>
+
+	* gdb.base/break-interp.exp (test_ld) <istarget powerpc64-*>: Add
+	is_lp64_target to check 64bit target.
+
 2010-09-09  Jan Kratochvil  <jan.kratochvil@redhat.com>
 
 	* gdb.base/break-interp.exp (reach): Permit leading . for ppc64.
--- src/gdb/testsuite/gdb.base/break-interp.exp	2010/09/09 20:00:49	1.19
+++ src/gdb/testsuite/gdb.base/break-interp.exp	2010/09/10 20:29:25	1.20
@@ -401,7 +401,7 @@
 	}
 	# `info sym' cannot be tested for .opd as the binary may not have
 	# symbols.
-	if [istarget powerpc64-*] {
+	if {[istarget powerpc64-*] && [is_lp64_target]} {
 	    set test "convert entry point"
 	    gdb_test_multiple "p *(void(*)(void) *) 0x$entrynohex" $test {
 		-re " =( \\(\[^0-9\]*\\))? 0x(\[0-9a-f\]+)( < \[^\r\n\]*)?\r\n$gdb_prompt $" {
